src/css/sidebar.css

```diff
@@ -75,4 +75,29 @@
 
 .icon {
     margin-right: 12px;
+}
+
+.comparison-title {
+    display: flex;
```

Comment:
This text should be black.

Padding should be 19px on top, 13px on bottom.

`align-items: stretch` appears to be unnecessary.